### ISSUE TITLE
[Bug] Problems uploading files

### ISSUE THREAD

**Issue by jamebal** [IceWhaleTech/CasaOS#448]
When the site is https, the uploaded file is http

As shown in the picture:
<img width="937" alt="截屏 27" src="https://user-images.githubusercontent.com/49510841/184472854-89c56550-2188-4292-b9f6-e4d4cdf43038.png">

**Comment by jerrykuku** [comment]
Yes, this is a bug, and we will fix it in vNext release. Thanks for the feedback.
